docs/appendices/EOL.rst

```diff
@@ -7,14 +7,13 @@ The release before that gets critical security updates only.
 Older releases are marked end of life and receive no updates at all.
 Pre-releases do not receive immediate security updates.
 
-The currently supported release train of PowerDNS Authoritative Server is 4.4.
+The currently supported release train of PowerDNS Authoritative Server is 4.5.
 
-PowerDNS Authoritative Server 4.3 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.5 is released.
-It will be end of life after PowerDNS Authoritative Server 4.6 is released.
+PowerDNS Authoritative Server 4.4 will only receive correctness, stability and security updates and will be receiving critical updates only after PowerDNS Authoritative Server 4.6 is released. It will be end of life after PowerDNS Authoritative Server 4.7 is released.
 
-PowerDNS Authoritative Server 4.2 will only receive critical security updates and will be end of life after PowerDNS Authoritative Server 4.5 is released.
+PowerDNS Authoritative Server 4.3 will only receive critical security updates and will be end of life after PowerDNS Authoritative Server 4.6 is released.
```

Comment:
you might want to leave "security" out here

Reply:
indeed, changed